src/features/offer/pages/Offer/Offer.tsx

```diff
@@ -54,7 +79,9 @@ export const Offer: FunctionComponent = () => {
         logConsultWholeOffer()
       }
       // The log event is triggered when the similar offer playlist is visible
-      if (isCloseToBottom({ ...nativeEvent, padding: 300 })) {
+      const hasTwoSimilarOffersPlaylist =
+        hasSameCategorySimilarOffers && hasOtherCategoriesSimilarOffers
+      if (isCloseToBottom({ ...nativeEvent, padding: hasTwoSimilarOffersPlaylist ? 600 : 300 })) {
```

Comment:
Tu devrais exporter ça dans une constante ou une fonction bien nommée :
```suggestion
const PLAYLIST_HEIGHT = 300;

function getPlaylistsHeight(numberOfPlaylists: number) {
  return PLAYLIST_HEIGHT * numberOfPlaylists
}
      if (isCloseToBottom({ ...nativeEvent, padding: getPlaylistsHeight(hasTwoSimilarOffersPlaylist ? 2 : 1) })) {
```